src/Nethermind/Nethermind.Network/StaticNodes/StaticNodesManager.cs

```diff
@@ -57,14 +57,29 @@ public async Task InitAsync()
 
             string data = await File.ReadAllTextAsync(_staticNodesPath);
             string[] nodes = GetNodes(data);
-            if (_logger.IsInfo) _logger.Info($"Loaded {nodes.Length} static nodes from file: {Path.GetFullPath(_staticNodesPath)}");
+            if (_logger.IsInfo)
+                _logger.Info($"Loaded {nodes.Length} static nodes from file: {Path.GetFullPath(_staticNodesPath)}");
             if (nodes.Length != 0)
             {
                 if (_logger.IsDebug) _logger.Debug($"Static nodes: {Environment.NewLine}{data}");
             }
-            
-            _nodes = new ConcurrentDictionary<PublicKey, NetworkNode>(nodes.Select(n => new NetworkNode(n))
-                .ToDictionary(n => n.NodeId, n => n));
+
+            IEnumerable<NetworkNode> networkNodes = new List<NetworkNode>();
+
+            foreach (var n in nodes)
+            {
+                try
+                {
+                    NetworkNode networkNode = new(n);
+                    networkNodes = networkNodes.Append(networkNode);
+                }
+                catch (Exception exception)
```

Comment:
Any exception here, not only ArgumentException, SocketException, NetworkException?

Reply:
Now I catch Argument Exception and SocketException, I think that's ok